src/1_api.js

```diff
@@ -51,12 +51,32 @@ Server.prototype.serializeObject = function(obj, prefix) {
  * @param {Object.<string, *>} data
  */
 Server.prototype.getUrl = function(resource, data) {
-	var k, v, err;
-	var url = resource.destination + resource.endpoint;
+	var k, v, err,
```

Comment:
we don't really need to declare k here, do we? Instead we can restrict its scope to the loops only.

Similarly, v can be downgraded to a local var on line 90